src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
instead of creating new Random here I would advise to create 'protected static final Random RANDOM = new Random();'. 'protected' modifier allows to use it also in Lottery class .Than use it to use nextInt method. Also make method 'getRandomColor()' non-static.